clock/log.go

```diff
@@ -0,0 +1,26 @@
+package clock
+
+import "github.com/btcsuite/btclog"
+
+// log is a logger that is initialized with no output filters.  This
+// means the package will not perform any logging by default until the caller
+// requests it.
+var log btclog.Logger
+
+// The default amount of logging is none.
+func init() {
+	DisableLog()
+}
+
+// DisableLog disables all library log output.  Logging output is disabled
+// by default until UseLogger is called.
+func DisableLog() {
+	UseLogger(btclog.Disabled)
+}
+
+// UseLogger uses a specified Logger to output package logging info.
+// This should be used in preference to SetLogWriter if the caller is also
+// using btclog.
+func UseLogger(logger btclog.Logger) {
+	log = logger
+}
```

Comment:
Log is unused? Also seems a bit too much maybe, to log for `clock`.

Reply:
I added the log because there's a debug log in `TestClock` (you added that actually). Would you like to remove it? (I too think logging is not necessary).

Reply:
Ah, that is right. Yes, I added the logging because it was quite useful to debug the tests. Maybe we could otherwise pass in a logger to `TestClock`? Or you can remove it if you don't like that.

As a side note, if you add a typical `log.go` file, it also needs to be connect in the root `log.go`.

Reply:
I'm removing it then as there's no need to debug (tests added).

Reply:
Ok. To clarify: I didn't add them to debug the test clock, but to debug the test that was using the test clock.